parsers/cobol.c

```diff
@@ -41,33 +49,120 @@ static kindDefinition CobolKinds[] = {
 
 static tagRegexTable cobolTagRegexTable[] = {
 	{ "......\\*.*", "", "", "{exclusive}" },
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "[FSR]D[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
 	 "f,fd,file descriptions (FD, SD, RD)", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "[0-9]+[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
 	 "g,group,group items", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "PROGRAM-ID\\.[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
 	 "P,program,program ids", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "([A-Z0-9][A-Z0-9-]*)[ \t]+SECTION\\.", "\\1",
 	 "s,section,sections", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "([A-Z0-9][A-Z0-9-]*)[ \t]+DIVISION\\.", "\\1",
 	  "D,division,divisions", "i"},
+	{ COBOL_REGEX_PREFIX_RELAXED
```

Comment:
I think if we introduce COBOL_REGEX_PREFIX_UNIFIED, these changes are not needed.